wgpu-core/Cargo.toml

```diff
@@ -38,8 +38,13 @@ raw-window-handle = { version = "0.3", optional = true }
 ron = { version = "0.5", optional = true }
 serde = { version = "1.0", features = ["serde_derive"], optional = true }
 smallvec = "1"
+spirv_headers = { version = "1.4.2" }
 vec_map = "0.8"
 
+[dependencies.naga]
+git = "https://github.com/gfx-rs/naga"
```

Comment:
since `wgpu-core` is not a binary target, this should specify the exact revision instead of just a branch.
For binary targets, just including "Cargo.lock" under version control solves it.